src/stacked_borrows.rs

```diff
@@ -338,18 +345,15 @@ impl<'tcx> Stack {
         &mut self,
         access: AccessKind,
         tag: SbTag,
-        dbg_ptr: Pointer<AllocId>, // just for debug printing amd error messages
+        (alloc_id, range, offset): (AllocId, AllocRange, Size), // just for debug printing amd error messages
```

Comment:
```suggestion
        (alloc_id, range, offset): (AllocId, AllocRange, Size), // just for debug printing and error messages
```
(old typo that I probably wrote.^^)

Somewhere (in the doc comment above?) it should explain that the `range` is for the *entire* access/reborrow, and `offset` is the location of the stack that we currently act on.